R/sparkR-mosaic/README.md

```diff
@@ -2,6 +2,31 @@
 
 sparkrMosaic is a lightweight extension of SparkR to expose Mosaic's large scale geospatial data processing functions.
 
-## Contributing to the R bindings for Mosaic
+## Getting started
+
+### Requirements
```

Comment:
This section has been moved to the main Readme. Could you please update that section instead?